docs/source/tutorials/psij.ipynb

```diff
@@ -0,0 +1,547 @@
+    "        attributes=JobAttributes(\n",
+    "            queue_name=\"<QUEUE_NAME>\", project_name=\"<ALLOCATION>\"\n",
+    "        ),\n",
+    "    )\n",
+    ")\n",
+    "\n",
+    "print(str(job.status))\n",
+    "ex.submit(job)\n",
+    "job.wait()\n",
+    "print(str(job.status))\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: The <QUEUE_NAME> and <ALLOCATION> fields will depend on the system you are running on."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Managing Job State\n",
+    "In all the above examples, we have submitted jobs without checking on what happened to them. Once the job has finished executing (which, for */bin/date*, should be almost as soon as the job starts) the resource manager will mark the job as complete, triggering PSI/J to do the same via the [JobStatus](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_status.JobStatus) attribute of the job. `Job` state progressions follow this state model:\n",
+    "\n",
+    "<img src=\"./images/psij_job_states.png\" width=\"350\"/>\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Cancelling a Job\n",
+    "If supported by the underlying job scheduler, PSI/J jobs can be canceled by invoking the [cancel](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job.cancel) method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = Job(JobSpec(executable=\"/bin/date\"))\n",
+    "ex.submit(job)\n",
+    "job.cancel()\n",
+    "print(str(job.status))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Status Callbacks\n",
+    "Waiting for jobs to complete with [wait](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job.wait) is fine if you don’t mind blocking while you wait for a single job to complete. However, if you want to wait on multiple jobs without blocking, or you want to get updates when jobs start running, you can attach a callback to a [JobExecutor](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor) which will fire whenever any job submitted to that executor changes status.\n",
+    "\n",
+    "To wait on multiple jobs at once:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "from psij import Job, JobExecutor, JobSpec\n",
+    "\n",
+    "count = 3\n",
+    "\n",
+    "def callback(job, status):\n",
+    "    global count\n",
+    "\n",
+    "    if status.final:\n",
+    "        print(f\"Job {job} completed with status {status}\\n\")\n",
+    "        count -= 1\n",
```

Comment:
This may or may not work properly without synchronization. Probably fine to leave like this for tutorial.